tests/common.go

```diff
@@ -10948,3 +10950,41 @@ func SwitchBothKubeConfigANDContext(cluster string) error {
 	}
 	return nil
 }
+
+// DoPDBValidation continuously validates the Pod Disruption Budget against
+// cluster upgrades, appending errors to mError, until a stop signal is received.
+func DoPDBValidation(stopSignal <-chan struct{}, mError *error) {
+	pdbValue, allowedDisruptions := GetPDBValue()
+	isClusterParallelyUpgraded := false
+	nodes, err := Inst().V.GetDriverNodes()
+	if err != nil {
+		*mError = multierr.Append(*mError, err)
+		return
+	}
+	totalNodes := len(nodes)
+	itr := 1
+	for {
+		log.Infof("PDB validation iteration: #%d", itr)
+		select {
+		case <-stopSignal:
+			if allowedDisruptions > 1 && !isClusterParallelyUpgraded {
+				err := fmt.Errorf("cluster is not parallely upgraded")
+				*mError = multierr.Append(*mError, err)
+				log.Warnf("Cluster not parallely upgraded as expected")
+			}
+			log.Infof("Exiting PDB validation routine")
+			return
+		default:
+			errorChan := make(chan error, 50)
+			ValidatePDB(pdbValue, allowedDisruptions, totalNodes, &isClusterParallelyUpgraded, &errorChan)
```

Comment:
nit : Since ValidatePDB function is in the same file, you can change it to validatePDB

Reply:
Oh I assumed all cases will be written in this file itself. Otherwise we can keep it the same